WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -2,11 +2,9 @@ import SwiftUI
 
 struct PointOfSaleDashboardView: View {
     @EnvironmentObject private var posModel: PointOfSaleAggregateModel
-    @ObservedObject private var viewModel: PointOfSaleDashboardViewModel
 
-    init(viewModel: PointOfSaleDashboardViewModel) {
-        self.viewModel = viewModel
-    }
+    @State private var showExitPOSModal: Bool = false
```

Comment:
👍 Given this state var is only used to reflect user selection within the dashboard to open the modal, I think using '@State + @Binding` is a good move. Rather than using a view model as an intermediary.